core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/JavascriptPluginInclusion.java

```diff
@@ -53,6 +53,7 @@
 public class JavascriptPluginInclusion {
 
   private static final String javascriptPath = URLUtil.getApplicationURL() + "/util/javaScript/";
+  private static final String flashPath = URLUtil.getApplicationURL() + "/util/flash/";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)